libs/python.json

```diff
@@ -14,6 +14,7 @@
   {"id":"py.PyYAML","imports":["PyYAML"],"name":"PyYAML","repo":"yaml/pyyaml","tags":["yaml-parser"],"tech":["app-data"],"status":"awaiting-model"},
   {"id":"py.lxml","imports":["lxml"],"name":"lxml","repo":"lxml/lxml","tags":["xml-parser"],"tech":["app-data","http://lxml.de/"],"status":"awaiting-model"},
   {"id":"py.marshmallow","imports":["marshmallow"],"name":"marshmallow","repo":"marshmallow-code/marshmallow","tags":["serialization"],"tech":["app-data"],"status":"awaiting-model"},
+  {"id":"py.watchdog","imports":["watchdog."],"name":"watchdog","repo":"gorakhargosh/watchdog","tags":["filesystem"],"tech":["app-data"],"status":"awaiting-model"},
```

Comment:
This import looks funny. Maybe just 'watchdog'?

Reply:
I can change if you like. It seems imports always go with subnames like watchdog.something. I saw bunch of examples where dot is used (grep -E 'imports"[:]\["\w+\."' *) so I used it here as well. Not 100% sure whether this is correct

Reply:
yeah, I think you should change them all. There is nothing wrong with importing whole modules. It's a matter of habit, style, preference, or an org policy.